from events.domain.exceptions.base import DomainError


class AttendeeAlreadyExistsError(DomainError):
    ...
